lib/version.h

```diff
@@ -2,8 +2,8 @@
 // After a release the DEVMINOR is incremented. MAJOR=x MINOR=y, DEVMINOR=y+1
 
 #define CPPCHECK_MAJOR 2
-#define CPPCHECK_MINOR 9
```

Comment:
I believe there is a misunderstanding here.
Latest stable release is 2.10
Therefore the dev version should be 2.10.99
So CPPCHECK_MINOR should be 10
And CPPCHECK_DEVMINOR should be 11
Then the CPPCHECK_VERSION_STRING will become "2.11 dev"

Reply:
It seems the intention with DEVMINOR was to hold the minor version under development.

Reply:
That would make more sense but then then the dev version would always have a higher version with the 2.10.x series and would seems like a newer version to use.

I would propose something I have used before a simpler approach to what GCC and Clang are doing.

x.y.0 - the dev version - 2.11.0
x.y.1 - the first stable release - 2.11.1
x.y.z  - the following patch versions 2.11.2 etc.
x.a.0 - the next dev version - 2.12.0

As the `.0`is defined as the dev version there's no separate indicator necessary for it at all. It would also not break the parsable version by adding a string. It's also very straight forward to document and understand as it has precedence.

Integrating the commit hash for each more precise versions would also be possible.

There's the possibly of using the `-PRERELEASE` (do not remember 100% - need to look it up again) suffix which would be a valid parsable version. But for that we need a proper scripting approach - which should be easy work though (as I have done that before). Hence why I wanted to propose this *after* I reviewed the release process.